kale/loaddata/digits_access.py

```diff
@@ -68,7 +68,7 @@ class DigitDatasetAccess(DatasetAccess):
     """Common API for digit dataset access
 
     Args:
-        data_path (string): root directory of dataset
+        data_path (string): Whether sort images
```

Comment:
data_path to indicate whether to sort images? A bit confused. More explanation please.

Reply:
It was changed mistakenly. Will change it back.